tests/nightly/nightly-autorun.sh

```diff
@@ -16,8 +16,9 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
-git reset --hard origin/master
+sudo git clean -fd
```

Comment:
We should address this point before merging this change, since those nightly text log files are useful for debugging failures.